python-macos/config.sh

```diff
@@ -21,37 +21,66 @@ set -e
 
 function build_wheel {
     echo `pwd`
+    export PATH="/usr/local/opt/flex/bin:/usr/local/opt/bison/bin:$PATH"
```

Comment:
I don't think this will make `cmake` find the executable here. Does `cmake` have it's own list of paths that it searches for executables that doesn't necessarily respect `PATH`?

Reply:
Hm, it looks like `PATH` *is* eventually searched: https://cmake.org/cmake/help/v3.10/command/find_program.html

Reply:
Locally this fixed the issue for me which makes it harder to debug.